internal/daemon/cluster/handlers/worker_service.go

```diff
@@ -333,6 +333,7 @@ func (ws *workerServiceServer) LookupSession(ctx context.Context, req *pbs.Looku
 		Authorization: &targets.SessionAuthorizationData{
 			SessionId:   sessionInfo.GetPublicId(),
 			Certificate: sessionInfo.Certificate,
+			PrivateKey:  sessionInfo.CertificatePrivateKey,
```

Comment:
do we want to assert this in any of the unit tests?

Reply:
Good spot, I added asserts for this value.